src/SprykerMiddleware/Zed/Process/Business/ProcessFacade.php

```diff
@@ -35,4 +35,30 @@ public function translate(array $payload, array $dictionary)
             ->createTranslator($dictionary)
             ->translate($payload);
     }
+
+    /**
+     * @param array $payload
+     * @param string $destination
+     *
+     * @return array
+     */
+    public function writeSerialized(array $payload, string $destination)
```

Comment:
i would be better to have single write() method and determine format with parameters